doc-content/enterprise-only/openshift/proc-dashbuilder-standalone.adoc

```diff
@@ -0,0 +1,77 @@
+[id="proc-dashbuilder-standalone_{context}"]
+= Installing Dashbuilder Standalone using Operator
+
+You can use Operator to install Dashbuilder Standalone independently from other services.
+
+.Procedure
+
+. On the *Installation* page, in the *Application name* field, enter `standalone-dashbuilder`.
+. In the *Environment* field, enter `rhpam-standalone-dashbuilder`.
+. Click *Next*.
+. On the *Security* page, click *Next*.
+. On the *Components* page, from the *Components* list, select *Dashbuilder*.
```

Comment:
Similar to the previous suggestion. Not show-stoppers, just my approach.

On the *Components* page, select *Dashbuilder* from the *Components* list.

Reply:
Done